src/pages/PayPage/CreateRequestPage.tsx

```diff
@@ -0,0 +1,187 @@
+import { BottomBtn } from "@/components/BottomBtn";
+import { useState } from "react";
+import CompletePay from "./CompletePay";
+import CompleteCreatePay from "./CompleteCreatePay";
+import CheckBox from "@/components/CheckBox";
+import { Member } from "../ChatPage/ChatCreatePage/ChatCreatePage.styled";
+import ReactImg from "@/assets/react.svg";
+import { PayChatDiv } from "./CreatePayComponents";
+import SearchIcon from "@/assets/PayPage/search_icon.svg";
+
+const RecentAccountDiv = () => {
+  return (
+    <s.RowFlexDiv style={{ margin: "0.25rem" }}>
+      <img style={{ marginRight: "0.75rem" }} src={Kookmin}></img>
+      <s.ColumnFlexDiv>
+        <s.GrayTextDiv>국민은행</s.GrayTextDiv>
+        <s.RegularText>123-1234-12345</s.RegularText>
+      </s.ColumnFlexDiv>
+    </s.RowFlexDiv>
+  );
+};
+
+type NextPageType = {
+  nextPage: Function;
+};
+const CreateRequestPage1 = ({ nextPage }: NextPageType) => {
+  return (
+    <>
+      <TopBarText left={LeftEnum.Back} center="" right=""></TopBarText>
+      <s.ContainerDiv>
+        <div style={{ margin: "2rem 0rem 2rem 0rem" }}>
+          <s.NowPriceDiv>정산받을 계좌를 입력해주세요</s.NowPriceDiv>
+        </div>
+        <div style={{ margin: "2rem 0rem 3rem 0rem" }}>
+          <s.InputText placeholder="계좌번호"></s.InputText>
+          <s.BankSelect>
+            <s.BankOption disabled hidden selected>
+              은행 선택
+            </s.BankOption>
+            <s.BankOption value="kookmin">국민은행</s.BankOption>
+            <s.BankOption value="shinhan">신한은행</s.BankOption>
+          </s.BankSelect>
+        </div>
+        <div style={{ margin: "2rem 0rem 2rem 0rem" }}>
+          <s.RegularText>최근 정산받은 계좌</s.RegularText>
+          <s.RoundDiv>
+            <RecentAccountDiv></RecentAccountDiv>
+          </s.RoundDiv>
+        </div>
+      </s.ContainerDiv>
+      <BottomBtn onClick={() => nextPage()}>다음으로</BottomBtn>
+    </>
+  );
+};
+const CreateRequestPage2 = ({ nextPage }: NextPageType) => {
+  const [tabIndex, setTabIndex] = useState(0);
+  const [search, setSearch] = useState("");
+
+  const onKeywordChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    console.log(search);
+    setSearch(e.target.value);
+  };
+
+  const selectMenuHandler = (index: number) => {
+    setTabIndex(index);
+    console.log(index);
+  };
+  const menuArr = [
+    { name: "채팅방", content: "Tab menu ONE" },
+    { name: "멤버", content: "Tab menu TWO" },
+  ];
+  const dataArr = [
+    { image: "aa", name: "박규환", amount: "15000원", chk: true },
+    { image: "aa", name: "박규환", amount: "15000원", chk: true },
+  ];
+  const chatroomArr = [
+    { img: "aa", name: "GreenJoa 방", cnt: 37 },
+    { img: "aa", name: "작업 안 하면 죽는 방", cnt: 24 },
+  ];
+
+  return (
+    <>
+      <TopBarText left={LeftEnum.Back} center="" right=""></TopBarText>
+      <s.ContainerDiv>
+        <s.NowPriceDiv>정산할 멤버를 선택해주세요</s.NowPriceDiv>
+        <s.TabMenu>
+          {menuArr.map((value, index) => (
+            <li
```

Comment:
```suggestion
            <li key={index}
```

map 사용할 때는 자식 component한테 꼭 index 속성 넣어주세요~!
안그러면 console err가 뜹니다..

Reply:
갓진호 강의에서도 그 부분이 있긴했는데 뭔가 console에 warning으로 떠서 매번 무시했었네요
그렇게 수정하겠습니다.